### PR TITLE
Update Baseline table look

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: src/SmartComponents/BaselinesTable/BaselinesTable.js [hunk 1]
```diff
@@ -55,13 +60,17 @@ class BaselinesTable extends Component {
...
+        const { baselineTableData, addBaselineUUID } = this.props;
         let tableWithKebab = [];
 
         baselineTableData.forEach(function(baseline) {
             let row = [];
-            baseline.forEach(function(data) {
-                row.push(data);
+            baseline.forEach(function(data, index) {
+                if (index === 0) {
+                    row.push(<div><a onClick={ addBaselineUUID.bind(this, baseline.id) }>{ data }</a></div>);
```
Comment (ehelms) [inline src/SmartComponents/BaselinesTable/BaselinesTable.js]: @johnsonm325 Wondered if you had any ideas here; this onClick won't render in the table. I assume it is due to some processing being done by the component that consumes these rows.

**Comment by ehelms** [comment]
This aims to address some of the required UI changes. The click to edit is not working yet.

**Comment by beav** [comment]
I get the "no baselines" page along with baselines

image

**Comment by ehelms** [comment]
Oops you found my leftover test condition. Fixed

**Pull request merged**